CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/ktruss/NestedKTrussDisplayPanel.java

```diff
@@ -123,12 +123,18 @@ private int rectX(final int rectangleNum) {
     // Returns the y position on this panel of the given rectangle.
     private int rectY(final int rectangleNum) {
         final int y = rectangles[rectangleNum][2];
+        if(totalNeededHeight == 0){
+            return y;
+        }
         return expandHeight ? (y * PREFERRED_HEIGHT) / totalNeededHeight : y;
```

Comment:
Given the return values here, could you potentially add your added if condition to the ternary below it?
i.e. `return expandHeight && totalNeededHeight > 0 ? ... : y`
I'm assuming here that `totalNeededHeight` isn't going negative but could also be changed from `>` to `!=` if need be